.github/workflows/ets-from-source.yml

```diff
@@ -0,0 +1,52 @@
+name: Test with EDM using ETS packages from source
```

Comment:
When using the `gh` tool from the command line it's possible to refer to workflows by name. I'm wondering whether it's worth using a more snappy name here for that purpose. (It's not essential, since one can also refer to workflows by their filename.)